test/single_server_tests.js

```diff
@@ -187,5 +187,33 @@ describe('Server', function() {
         yield server.stop();
       });
     });
+
+    it('test starting server instance with logpath', function() {
+      return co(function*() {
+        var Server = require('../').Server;
+
+        // Create dbpath
+        var dbpath = f('%s/../db', __dirname);
+
+        // Create dbpath
+        var logpath = f('%s/../output.log', __dirname);
+
+        // Create new instance
+        var server = new Server('mongod', {
+          dbpath: dbpath,
+          logpath: logpath
+        });
+        managers.push(server);
+
+        // Purge the directory
+        yield server.purge();
+
+        // Start process
+        yield server.start();
+
+        // Stop the process
+        yield server.stop();
```

Comment:
it doesn't appear that anything is asserted in this test

Reply:
There isn't any assertion to be made. The issue that occurs in this test is that `server.start()` hangs forever because it's waiting for output in `stdout` when it's all going to the log.

Not sure what assertion should be made? Should I make a note about this?